sig-release/README.md

```diff
@@ -0,0 +1,122 @@
+# SIG Release
+
+### Table of Contents
+- [Purpose](#purpose)
+- [Join us!](#join-us)
+- [Broad responsibilities](#broad-responsibilities)
+- [Specific responsibilities](#specific-responsibilities)
+- [Relation to the Release Manager and Team](#relation-to-the-release-manager-and-team)
+- [Ongoing responsibilities of the Release Team](#ongoing-responsibilities-of-the-release-team)
+  - ["Minor" Releases](#minor-releases)
+  - [“Patch” releases](#patch-releases)
+  - ["Security" releases](#security-releases)
+- [References](#references)
+
+## Join us!
+- [Google group](https://groups.google.com/forum/#!forum/kubernetes-sig-release)
+- [Slack channel](https://kubernetes.slack.com/messages/C56HWQ0TH/)
+- [Events and meetings calendar](https://calendar.google.com/calendar/embed?src=coreos.com_regcvcrgvq98lua2ikijg1g1uk%40group.calendar.google.com&ctz=America/Los_Angeles)
+
+## Purpose
+- Production of high quality Kubernetes releases on a reliable schedule
+- Ensure there is a [consistent group of community members](https://docs.google.com/document/d/1JAUqKl-lYdYLQ7GUT_9LzqxwQv-PcOdyAxNRZKItajo/edit) [1] in place to support the release process
+- Provide guidance and tooling to facilitate the production of automated releases
+- Serve as a tightly integrated partner with other SIGs to integration their repositories into the release process
```

Comment:
The overall purpose/goal here is to ensure (early) collaboration and integration (tooling) with any release-critical repos outside of core kubernetes/kubernetes.  I'm not sure we need to spell out that list here and now as there are no other repos that currently fall into this category?  Maybe kubernetes/features for .0 releases.  
Providing a more generalized release process is certainly part of this.